proxy/pkg/middleware/oidc_auth.go

```diff
@@ -96,16 +96,12 @@ func (m oidcAuth) getClaims(token string, req *http.Request) (claims oidc.Standa
 			return
 		}
 
-		// TODO allow extracting arbitrary claims ... or require idp to send a specific claim
 		if err := userInfo.Claims(&claims); err != nil {
 			m.logger.Error().Err(err).Interface("userinfo", userInfo).Msg("failed to unmarshal userinfo claims")
 			status = http.StatusInternalServerError
 			return
 		}
 
-		//TODO: This should be read from the token instead of config
```

Comment:
- [x] ~~it may not be in the token?~~ the spec says it must be: https://openid.net/specs/openid-connect-core-1_0.html#IDToken